sysid-application/src/main/native/include/sysid/analysis/FeedbackControllerPreset.h

```diff
@@ -31,15 +31,23 @@ struct FeedbackControllerPreset {
   /** The measurement delay in the encoder measurements. */
   units::second_t measurementDelay;
 
-  /** Checks equality between two feedback controller presets. */
+  /** Checks equality between two feedback controller presets.
+   * @param rhs Another FeedbackControllerPreset
+   *
+   * @return If the two presets are equal
+   */
```

Comment:
This applies to other comments too.
```suggestion
  /**
   * Checks equality between two feedback controller presets.
   *
   * @param rhs Another FeedbackControllerPreset
   * @return If the two presets are equal
   */
```